src/DigitalTwins/Changelog.md

```diff
@@ -0,0 +1,24 @@
+<!--
+    Please leave this section at the top of the change log.
+
+    Changes for the upcoming release should go under the section titled "Upcoming Release", and should adhere to the following format:
+
+    ## Upcoming Release
+    * Overview of change #1
+        - Additional information about change #1
+    * Overview of change #2
+        - Additional information about change #2
+        - Additional information about change #2
+    * Overview of change #3
+    * Overview of change #4
+        - Additional information about change #4
+
+    ## YYYY.MM.DD - Version X.Y.Z (Previous Release)
+    * Overview of change #1
+        - Additional information about change #1
+-->
+## Upcoming Release
+
+## Version 0.1.0
+* First preview release for module Az.DigitalTwins
```

Comment:
Please do not add versions in change log. You should append new change logs right below "Upcoming release"
```suggestion
## Upcoming Release
* First preview release for module Az.DigitalTwins
```

Reply:
Got it ,thanks.